hooks/textbook-toc-utils.ts

```diff
@@ -19,6 +16,10 @@ function extractToc (indexContent: string): TocType {
   function getTopics (index: number, allTopics: string[]) {
     return allTopics.filter(topic => topic.startsWith(`${index}.`))
   }
+
+  function clean (str: string) {
+    return str.replace(/>\s+/, '').replace(/\s+/g, ' ').trim()
```

Comment:
I think we can improve the readability of this function if we separate responsibilities and create a function for each step of the "clean"

Reply:
I'm clarifying it in form of a couple of comment. Happy to split into functions if this gets more complicated.